metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala

```diff
@@ -1252,6 +1252,7 @@ class MetalsLspService(
   def didChangeWatchedFiles(
       events: List[FileEvent]
   ): Future[Unit] = {
+    scribe.debug(s"did change watched files event $events")
```

Comment:
This could cause a lot of debug messages, no?

Reply:
Yeah, sorry forgot to delete this.